<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-DIMGNTL" failures="2" tests="14" name="com.dsalgo.tests.DSAlgoSignInTest" time="3.485" errors="0" timestamp="2022-09-07T12:26:44 EDT" skipped="12">
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="ValidateDropdownBeforeSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- ValidateDropdownBeforeSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="VerifySuccessfulSignOut" time="0.000">
    <skipped/>
  </testcase> <!-- VerifySuccessfulSignOut -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="VerifySuccessfulSignOut" time="0.000">
    <skipped/>
  </testcase> <!-- VerifySuccessfulSignOut -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="VerifyDSValuesinDropDownBeforeSignIn" time="0.001">
    <skipped/>
  </testcase> <!-- VerifyDSValuesinDropDownBeforeSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="VerifyDSValuesinDropDownBeforeSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyDSValuesinDropDownBeforeSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="ValidateGetStartedBeforeSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- ValidateGetStartedBeforeSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="NavigateToHomePage" time="1.152">
    <failure message="expected [Numpy Ninja] but found [NumpyNinja]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Numpy Ninja] but found [NumpyNinja]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.dsalgo.tests.DSAlgoSignInTest.NavigateToHomePage(DSAlgoSignInTest.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- NavigateToHomePage -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="ValidateGetStartedBeforeSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- ValidateGetStartedBeforeSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="RegisterSignInUser" time="0.000">
    <skipped/>
  </testcase> <!-- RegisterSignInUser -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="NavigateToHomePage" time="2.332">
    <failure message="expected [Numpy Ninja] but found [NumpyNinja]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Numpy Ninja] but found [NumpyNinja]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.dsalgo.tests.DSAlgoSignInTest.NavigateToHomePage(DSAlgoSignInTest.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:471)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- NavigateToHomePage -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="RegisterSignInUser" time="0.000">
    <skipped/>
  </testcase> <!-- RegisterSignInUser -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="FailedSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- FailedSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="FailedSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- FailedSignIn -->
  <system-out/>
  <testcase classname="com.dsalgo.tests.DSAlgoSignInTest" name="ValidateDropdownBeforeSignIn" time="0.000">
    <skipped/>
  </testcase> <!-- ValidateDropdownBeforeSignIn -->
  <system-out/>
</testsuite> <!-- com.dsalgo.tests.DSAlgoSignInTest -->
